pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -71,16 +71,12 @@ const (
 	launcherSuffix      = "-launcher"
 	workerSuffix        = "-worker"
 	gpuResourceName     = "nvidia.com/gpu"
-	cpuResourceName     = "cpu"
 	labelGroupName      = "group_name"
 	labelMPIJobName     = "mpi_job_name"
 	labelMPIRoleType    = "mpi_role_type"
 )
 
 const (
-	// SuccessSynced is used as part of the Event 'reason' when an MPIJob is
-	// synced.
-	SuccessSynced = "Synced"
```

Comment:
Where do we use the const before? I did not see any deletion about it in the PR.

Reply:
We use this const to record `SuccessSynced`  event for mpijob and I think `synced successfully` is not worth a event and I delete this line in pr https://github.com/kubeflow/mpi-operator/pull/120

```
	c.recorder.Event(mpiJob, corev1.EventTypeNormal, SuccessSynced, MessageResourceSynced)
```

Reply:
LGTM